sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/stream/DataStreams.java

```diff
@@ -48,14 +48,6 @@ public static InputStream inbound(Iterator<ByteString> bytes) {
     return new Inbound(bytes);
   }
 
-  /**
-   * Converts a single {@link OutputStream} into multiple {@link ByteString}s.
-   */
-  public static OutputStream outbound(CloseableThrowingConsumer<ByteString> consumer) {
```

Comment:
I would like to keep this TODO somehow, you can can the signature of the method so we can keep it.

Reply:
Done.